core/templates/domain/story/StorySummaryObjectFactory.ts

```diff
@@ -20,14 +20,14 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
-interface IStorySummaryBackendDict {
+export interface IStorySummaryBackendDict {
   'id': string;
   'title': string;
   'node_titles': string[];
   'thumbnail_filename': string;
   'thumbnail_bg_color': string;
   'description': string;
-  'story_is_published': boolean;
+  'story_is_published'?: boolean;
```

Comment:
Why is this optional, it doesn!t really make sense in case of boolena value.

Reply:
I thought this wasn't coming in the response in the topic view. On investigating, I found it was coming under the name `published`, So, I renamed it to `story_is_published` because the response names should not differ like this.

Reply:
Just to clarify, it was optional since this was a required field in the topic editor, to show the status in the list, but not in the topic viewer, since only published stories would be shown there anyway. 
So, renaming the backend value to use the correct field name sg!